server/tests/test_precall_webhook.py

```
@@ -0,0 +1,69 @@
+This helps debug the inbound call implementation.
+"""
+
+import requests
+import json
+
+# Your ngrok URL
+BASE_URL = "https://feb7de50d255.ngrok-free.app"
+PRE_CALL_URL = f"{BASE_URL}/api/v1/webhooks/vapi-pre-call"
+
+# Vapi webhook secret from your .env
+WEBHOOK_SECRET = "Ram786Rao876Sal999Boob123"
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "WEBHOOK_SECRET"

(S105)